azure-pipelines.yaml

```diff
@@ -63,8 +66,8 @@ stages:
     displayName: 'Validate Infra'
     jobs: 
       - job: vh_tenant_replacement
-        pool:
-          vmImage: 'ubuntu-latest'
+        # pool:
```

Comment:
can you remove the commented out text please as it is not required.

can you do that for all files please?

Reply:
Thats the commeneted out text removed